src/Entities/DreamBlocks/DreamMoveBlock.cs

```diff
@@ -56,8 +63,28 @@ public enum MovementState
 
     private Player noSquish;
 
+    private List<Image> topButton = new List<Image>();
+
+    private List<Image> leftButton = new List<Image>();
+
+    private List<Image> rightButton = new List<Image>();
```

Comment:
nitpick: spacing & syntax
```suggestion
    private List<Image> topButton = new();
    private List<Image> leftButton = new();
    private List<Image> rightButton = new();
```